pkg/ccl/backupccl/testdata/backup-restore/restore-grants

```diff
@@ -159,7 +159,7 @@ BACKUP INTO 'nodelocal://0/test/'
 # above.
 subtest cluster-restore
 
-new-server name=s2 share-io-dir=s1 allow-implicit-access
+new-server name=s2 share-io-dir=s1 allow-implicit-access disable-tenant
```

Comment:
Let's add a TODO with the issue you filed.

Reply:
done.